website/src/main/tut/docs/rules/SingleAbstractMethod.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: docs
+title: SingleAbstractMethod
+---
+
+# SingleAbstractMethod
+
+http://www.scala-lang.org/files/archive/spec/2.12/06-expressions.html#sam-conversion
+
+"-P:semanticdb:members:all",
```

Comment:
I think we can wait documenting the installation since we can change sbt-scalafix to configure the default flags so SAM works out of the box. `members:all` won't be necessary either once my Scope PR gets merged, same for `denotations:all`. I suspect we'll make overrides enabled by default